.github/workflows/native-cron-build.yml.disabled

```diff
@@ -59,18 +59,14 @@ jobs:
 
       - name: Report
         if: always()
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
-          STATUS: ${{ job.status }}
-          JAVA_VERSION: ${{ matrix.java }}
+        shell: bash
         run: |
-          echo "The report step got status: ${STATUS}"
-          sudo apt-get update -o Dir::Etc::sourcelist="sources.list" \
-            -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
-          sudo apt-get install -y gnupg2 gnupg-agent
-          echo Installing SDKMAN
-          curl -s "https://get.sdkman.io" | bash
-          source ~/.sdkman/bin/sdkman-init.sh && \
-          sdk install jbang 0.21.0
           [[ ${JAVA_VERSION} = 8 ]] && ISSUE_NUMBER="6717" || ISSUE_NUMBER="6723"
```

Comment:
Given we don't support JDK 8 anymore, we could simplify further, couldn't we?

Reply:
I was a bit on the fence about this one but yeah, dropping this predicate makes sense.